cumulus/parachains/runtimes/assets/test-utils/src/test_cases_over_bridge.rs

```diff
@@ -345,9 +347,9 @@ pub fn receive_reserve_asset_deposited_from_different_consensus_works<
 	XcmConfig: xcm_executor::Config,
 	LocationToAccountId: ConvertLocation<AccountIdOf<Runtime>>,
 	<Runtime as pallet_assets::Config<ForeignAssetsPalletInstance>>::AssetId:
-		From<MultiLocation> + Into<MultiLocation>,
+		From<xcm::v3::Location> + Into<xcm::v3::Location>,
 	<Runtime as pallet_assets::Config<ForeignAssetsPalletInstance>>::AssetIdParameter:
-		From<MultiLocation> + Into<MultiLocation>,
+		From<xcm::v3::Location> + Into<xcm::v3::Location>,
```

Comment:
@franciscoaguirre it would be very cool, if it would be possible to use only `latest` for these generic test-cases,
at least few lines above `(xcm::v3::Location, u128, u128),` could be change to latest.

and for:
```
<Runtime as pallet_assets::Config<ForeignAssetsPalletInstance>>::AssetIdParameter:
		From<xcm::v3::Location> + Into<xcm::v3::Location>,
```
hmm, I don't know, maybe some type-harakiri could do it, but it is just a blind guess. I added it to my TODOs to check later.